Assets/MRTK/SDK/Experimental/Drawstring/Scripts/ElasticHandleLogic.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Physics
+{
+
+    /// <summary>
+    /// Implements a two-handle elastic "stretch" logic, which allows for
+    /// either one or two pointers to stretch along a particular axis.
+    /// 
+    /// Built around the differential equations for a damped harmonic oscillator.
+    /// Intended for use with critically and over-damped oscillators.
+    /// 
+    /// Usage:
+    /// When a manipulation starts, call Setup.
+    /// Call Update any time to update the move logic and get a new rotation for the object.
+    /// </summary>
+    internal class ElasticHandleLogic
```

Comment:
why is this class internal?

Reply:
Discussed this with David; several other classes that are in this category are all `internal`, sticking with the status quo for now.